src/platform/xcode_templates.nit

```diff
@@ -528,7 +496,7 @@ class PlistTemplate
 	<key>CFBundleShortVersionString</key>
 	<string>{{{short_version}}}</string>
 	<key>CFBundleSignature</key>
-	<string>\\?\\?\\?\\?</string>
+	<string>?{{{"??"}}}?</string>
```

Comment:
It's a trigraph thing. because `??<` might be a trigraph. Another option is to add `-Wno-trigraphs` to the list of CC flags in `compiler/abstract_compiler.nit`